src/rp2_common/hardware_pio/include/hardware/pio.h

```diff
@@ -495,7 +495,7 @@ static inline void sm_config_set_clkdiv_int_frac(pio_sm_config *c, uint16_t div_
 static inline void pio_calculate_clkdiv8_from_float(float div, uint32_t *div_int, uint8_t *div_frac8) {
     valid_params_if(HARDWARE_PIO, div >= 1 && div <= 65536);
 #if PICO_PIO_CLKDIV_ROUND_NEAREST
-    div += 0.5f / 256; // round to the nearest 1/256
```

Comment:
I preferred the constant!

Reply:
:man_facepalming:  You're going to _love_ the additional commit that I've got coming soon then... :cold_sweat: